connecting-creating-collaborating.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Club Xata reunion: teamwork across borders'
+description: 'Amidst breathtaking views of the Portuguese hilly landscape and early summer sun, we recap our Spring 2024 offsite in Evora for connecting and collaborating with our colleagues.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/offsite-evora-2024/xata-team-evora.jpeg
+  alt: The Xata team off-site in Evora
+author: Lea Artemov
+date: 05-07-2024
+tags: ['company', 'culture']
+published: true
+slug: offsite-evora-2024
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/offsite-evora-2024/xata-team-evora.jpeg
+---
+
+![The Xata team in Evora](images/offsite-evora-2024/xata-team-evora.jpeg)
+
+In a post-pandemic world that is slowly returning to offices, at Xata we still believe in the power of remote work - we believe it empowers us as a team to do the best work possible for our product.
+
+But sometimes being assigned one colleague every other week for a virtual #donut-bot-meeting is not enough to build a strong and connected team if there's 30 people working remotely from 10 different countries across multiple continents.  
+That's why we get together twice a year for a week-long offsite to experience our team in real life instead of just on the screen.
+Most recently, we got a chance to meet in Evora, Portugal and spend 5 days taking in breathtaking views of the Portuguese hilly landscape and enjoying a bit of early summer sun.
+But offsites are not just a fun opportunity to vacation with your coworkers, they become an integral part of the company culture and leave a lasting impact for the remainder of the year.
+
+There's three key elements to every Club Xata Reunion - we come together to
+
+1. Collaborate in person
+2. Create unforgettable memories
+3. Connect authentically with our team
+
+Here's what that looks like for us:
+
+# Collaborating
+
+We love the benefits of remote work the rest of the year but nothing can quite replace actually sitting round a table with our team and brainstorming a new feature. That's why we dedicated a big chunk of our time in Evora to working together in person. We kicked off the working sessions with a vision meeting where Monica, our CEO, and other leaders shared their insights on where Xata is headed. We also got updates from different team leads, which helped everyone get on the same page and see the bigger picture instead of just the puzzle piece of our own individual work.
```

Comment:
```suggestion
We love the benefits of remote work the rest of the year but nothing can quite replace actually sitting around a table with our team and brainstorming a new feature. That's why we dedicated a big chunk of our time in Evora to working together in person. We kicked off the working sessions with a vision meeting where Monica, our CEO, and other leaders shared their insights on where Xata is headed. We also got updates from different team leads, which helped everyone get on the same page and see the bigger picture instead of just the puzzle piece of our own individual work.
```